doc/source/changelog.rst

```diff
@@ -9,13 +9,59 @@ Contributors
 ------------
 
 The `CREDITS file
-<https://github.com/yt-project/yt_astro_analysis/blob/master/CREDITS>`__
+<https://github.com/yt-project/yt_astro_analysis/blob/main/CREDITS>`__
 contains the most up-to-date list of everyone who has contributed to the
 yt_astro_analysis source code.
 
+Version 1.1
+-----------
+
+Release date: *December 9, 2021*
+
+New Features
+^^^^^^^^^^^^
+
+ * The HaloCatalog has been significantly refactored
+   `PR #58 <https://github.com/yt-project/yt_astro_analysis/pull/58>`__, `PR #62 <https://github.com/yt-project/yt_astro_analysis/pull/62>`__ with
+   the following additional improvements:
+
+     * the interface to the Rockstar halo finder is now compatible with the latest version of Rockstar Galaxies `PR #55 <https://github.com/yt-project/yt_astro_analysis/pull/55>`__
+     * all halo finders now support being run with time-series of datasets
+     * halo particle ids now savable with FoF and HOP halo finders `PR #52 <https://github.com/yt-project/yt_astro_analysis/pull/52>`__
+     * looping over halos is done with io chunks instead of ds.all_data for a significant speedup and reduction in memory
+     * Allow more flexibility for specifying rockstar particle mass `PR #84 <https://github.com/yt-project/yt_astro_analysis/pull/84>`__
+     * Add restart option for rockstar `PR #82 <https://github.com/yt-project/yt_astro_analysis/pull/82>`__
+     * Adding an outer_radius parameter to the iterative COM callback `PR #34 <https://github.com/yt-project/yt_astro_analysis/pull/34>`__
+
+ * Remove the sunyaev_zeldovich analysis module. This is now `pyxsim <http://hea-www.cfa.harvard.edu/~jzuhone/pyxsim/>`__. `PR #79 <https://github.com/yt-project/yt_astro_analysis/pull/79>`__
```

Comment:
This has not been moved to pyxsim but to https://github.com/jzuhone/ytsz.

Reply:
thanks John !